### ISSUE TITLE
Non Dfi balances

### ISSUE THREAD

**Issue by jonho79** [DeFiCh/app#969]
Hi,  I recently updated to 2.5.0 and completed synchronized the block chain, but my balances for USDT, ETH, BTC and 2 LP pools are not showing the correct amount (they are lower than early June when I last checked).  The transactions history for non DFI tokens is also dated as ETH is from Mar, BTC from Apr and USDT from Jan.  I also reindexed already and this did not solve the issue.  Prior to this recent issue, I was unable to start the blockchain for 2.5.0 as well as the prior 2.4 updates so that's why I had not checked since early June.

**Comment by izzycsy** [comment]
Hi @jonho79 , can you update to [v2.5.1](https://github.com/DeFiCh/app/releases/tag/v2.5.1), and if issue persist, can you try another `wallet.dat`? I'll provide the steps. You should be able to find other wallet.dat in the `wallets` folder, unless you've stored elsewhere on your PC. 

For Windows, the wallets folder is located here:
C: `\Users\<username>\AppData\Roaming\DeFi Blockchain\wallets`

For Mac — Go to Folder:  
`/Users/<username>/Library/Application Support/Defi/wallets`

In the wallets folder you'll likely see wallet-bak-123456789 as well, these are additional backups.

<img width="822" alt="wallet dat location" src="https://user-images.githubusercontent.com/65014479/122271024-894f4780-cf11-11eb-9d6a-247e85c9b937.png">

1. In the app, `Exit Wallet` 
<img width="236" alt="Exit Wallet" src="https://user-images.githubusercontent.com/65014479/122268801-18a72b80-cf0f-11eb-9a2c-7a0f36595111.png">

2. `Restore Wallet via Backup`
Select another wallet.dat in the `wallets` folder or where you've specified your other backups. 

<img width="1021" alt="Restore wallet from backup" src="https://user-images.githubusercontent.com/65014479/122268837-23fa5700-cf0f-11eb-9dd8-54f108ca8476.png">

**Comment by jonho79** [comment]
Hi,

I've downloaded 2.5.1 and 2.5.2 and I still don't see the correct balances
for anything but dfi token.   What should I do next?

**Comment by izzycsy** [comment]
Hi @jonho79 , did you try another `wallet.dat`? If your balance is correct but the tx history is missing then likely, you're facing [`missing tx history`](https://github.com/DeFiCh/app/issues/891) issue — this is a UI issue. Can you provide your logs for further investigation? And elaborate on what happened as it would help us understand what exactly is the issue. Thank you. 

#### Actual behavior:

#### How to reproduce it (as minimally and precisely as possible):

- DeFi Wallet Version: 
- Operating System (OS):
- Current free storage capacity:

Attach logs: 
<img width="339" alt="1 App log" src="https://user-images.githubusercontent.com/65014479/122738329-77c0c380-d2b4-11eb-96f7-722ac989ef4e.png">

<img width="511" alt="2 Compress" src="https://user-images.githubusercontent.com/65014479/122738343-7abbb400-d2b4-11eb-9b0f-3f5ff36f600e.png">

**Comment by jonho79** [comment]
I tried another wallet.dat and the issue persists.  Here are the logs
downloaded from right now.

I am using wallet 2.5.2 on Windows with 174 GB free.

The issue started when I was unable to start the blockchain starting with
2.5.0 (maybe a version before).  The solution was to uninstall and download
the wallet.  By doing this all the old back-up dat were deleted.  When I
reinstalled the wallet, the blockchain restarted and I started having the
issues with the wrong token balances for everything but DFI.   I downloaded
2.5.1 and 2.5.2 hoping for a fix but the balances are still
severely understated.  I have been unable to make any transactions for
close to 3 weeks so Im hoping I can resolve this soon.

I appreciate the help here and you folks help quite a bit when I first
started using in Dec and Jan and had issues.  It was running fairly
smoothly up until the updates in May.
